scraper/src/mindtouch2zim/processor.py

```diff
@@ -232,6 +235,18 @@ def run(self) -> Path:
             Illustration_48x48_at_1=zim_illustration.getvalue(),
         )
 
+        # jinja2 environment setup
+        self.jinja2_env = Environment(
+            loader=FileSystemLoader(ROOT_DIR.joinpath("templates")),
+            autoescape=select_autoescape(),
+        )
+        self.libretexts_glossary_template = self.jinja2_env.get_template(
+            "libretexts.glossary.html.jinja2"
```

Comment:
I prefer to store templates with their final extension (here `.html`) as it allows editing them in good conditions. Those being in a templates/ folder, there's little room for confusion

Reply:
You're right, I assumed that `j.inja2` extension would help prettier and others, but I didn't checked, and I have to admit now that `.html` is better for prettier and in VSCode syntax highlighting of HTML document with Jinja2 loops / conditions still works well when `.html` is used. Changed to `.html`